exporter/exporter.go

```diff
@@ -142,17 +142,17 @@ func Start(client rpc.Client) error {
 		// Make sure that all blocks are correct by comparing all block hashes in the database to the ones we have in the node
 		head, err := client.GetChainHead()
 		if err != nil {
-			logger.Fatal(err)
+			utils.LogError("getting chain head from client error", err).Fatal()
 		}
 
 		dbBlocks, err := db.GetLastPendingAndProposedBlocks(1, head.HeadEpoch)
 		if err != nil {
-			logger.Fatal(err)
+			utils.LogError("getting proposed and pending blocks error ", err).Fatal()
```

Comment:
This very example made me think: We might want to log more information in scenarios like this. Adding the `head.HeadEpoch` information could help.

You could do that by using something like fmt.sprintf and add it as string into the `infoIdentifier`.

@recy21 had a different suggestion though:
Maybe use the `Variadic Functions` feature, see here: https://gobyexample.com/variadic-functions

What are your thoughts?

Reply:
You can now add an arbitrary amount of additional info strings to the log but this is not yet used by anything.